tardis/io/config_reader.py

```diff
@@ -232,7 +232,7 @@ def from_yaml(cls, fname, *args, **kwargs):
             yaml_dict, *args, **kwargs)
 
     @classmethod
-    def from_config_dict(cls, config_dict, validate=True, config_dirname=''):
+    def from_config_dict(cls, config_dict, validate=True, config_dirname='', plasma_only=False):
```

Comment:
I want to create a `custom config object` from saved `plasma attributes` in `HDF` file . I am creating this config , just from plasma attributes (not including `montecarlo` or `model` properties) , so that I can use it in `assemble_plasma` method.

**If I turn this switch** `ON` (`plasma_only=True`) , It will bypass checking `montecarlo` section (It checks for `convergence_strategy` under montecarlo) , which is not required for `assemble_plasma` method . If I do not introduce this switch , this function cannot return a `Configuration` object **based only on**  `Plasma attributes`. 

This is a workaround to reuse `assemble_plasma` method to create `Plasma` object from HDF file, and this switch will be required in this case only. Please read the 3rd paragraph of my description of PR, above for more info. 

Hope this satisfies your question.